vector/src/main/java/im/vector/activity/VectorMemberDetailsActivity.java

```diff
@@ -493,7 +493,7 @@ public void performItemAction(final int aActionType) {
             case ITEM_ACTION_KICK:
                 if (null != mRoom) {
                     enableProgressBarView(CommonActivityUtils.UTILS_DISPLAY_PROGRESS_BAR);
-                    mRoom.kick(mRoomMember.getUserId(), mRoomActionsListener);
+                    mRoom.kick(mRoomMember.getUserId(), "TODO: to replace after merge from Riot", mRoomActionsListener);
```

Comment:
I suggest keeping nil the kick reason in Tchap for the moment

Reply:
Talked about it IRL